whois-nrtm4/src/main/java/net/ripe/db/nrtm4/domain/ObjectData.java

```diff
@@ -0,0 +1,5 @@
+package net.ripe.db.nrtm4.domain;
+
+public record ObjectData(int objectId, int sequenceId) {
```

Comment:
LastWhoisObjects?
ObjectData is too ambiguous I think

Reply:
they're not always 'last', that's the thing. it's possible to do updates while this is running so an object might actually be deleted between getting List<ObjectData> and then getting the RPSL (which will then be in 'history'). "WhoisObjectMetaData"? "WhoisObjectState"? if so then what should RpslObjectData become?

Reply:
Maybe merge both? In the end is the same just with a new attribute 
Then we can call it WhoisObjectData?

Reply:
at the moment ObjectData and RpslObjectData are defined as records. records are immutable, so you can't do 'setRpslObject(...)'; all values must be passed in the constructor. i could use a normal POJO and leave the rpslObject field null until it gets filled but that's hacky -- better to keep the clean separation in definition and in the code, for my money. I'll make them WhoisObjectData and RpslObjectData for now.